src/collection.js

```diff
@@ -1159,6 +1161,27 @@ export class CollectionTransaction {
   constructor(collection, adapterTransaction) {
     this.collection = collection;
     this.adapterTransaction = adapterTransaction;
+
+    this._events = [];
+  }
+
+  _windEvent(action, payload) {
```

Comment:
I've never seen the word "wind"/"wound" used in this kind of context. It's only a private method so it's probably not a big deal. What do you think about "queued" or "buffered"?